README.md

```diff
@@ -211,4 +213,22 @@ A list of currently active Blockchains projects. Sorted Alphabetically.
 
   #### Tutorials
    - [Complete Guide to Full Stack Solana Development by Nader Dabit](https://dev.to/edge-and-node/the-complete-guide-to-full-stack-solana-development-with-react-anchor-rust-and-phantom-3291)
- - [How to Mint an NFT on Solana](https://www.quicknode.com/guides/web3-sdks/how-to-mint-an-nft-on-solana )
\ No newline at end of file
+ - [How to Mint an NFT on Solana](https://www.quicknode.com/guides/web3-sdks/how-to-mint-an-nft-on-solana )
+ ----------
+ ## &#127916; youtube tutorials >> you can explore about crypto, blockchain,DAOs,EVM many more to learn...... &#11015;&#65039;
+ 1. [Dapp university](https://youtu.be/uGPC9wNTBbw)
+ 2. [whiteboard crypto](https://youtu.be/mSeLVtBe2xE)
+ 3. [Ethereum foundation](https://youtu.be/TDGq4aeevgY)
+ 4. [ moralis web3](https://youtu.be/P87pLayUD8c) 
+ 5. [ clever programmer](https://youtu.be/Lz3m4BI1EfY)
+ ----------
+ 
+ ## &#128279; some discord servers checkout them >>>>>>> &#128309; &#128309; ![discord](https://i.pinimg.com/474x/aa/b1/b6/aab1b6322eafe103a7ff4f0f1daeb9c6.jpg)
+ 1.[Peeps Platform](https://discord.st/server/peeps-platform/)
+ 
+ 2.[Third Academy](https://discord.st/server/peeps-platform/)
```

Comment:
You have linked to Peeps Platform again

Reply:
Please take a look @JayrajSingh65 .

You can also remove the discord server section because the yt one looks great.

Reply:
hey,  I removed it hope now its looking cool check it out